ci/validate_wheel.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_dir=$1
+wheel_dir_relative_path=$2
+
+# TODO: pre-install these in ci-imgs
+rapids-logger "installing validation tools"
+python -m pip install \
+    -v \
+    'pydistcheck==0.8.0' \
+    'twine>=5.0.0'
+
+pyenv rehash
```

Comment:
TODO before merging: remove this after `pydistcheck` is added to ci-imgs: https://github.com/rapidsai/ci-imgs/pull/205

(`twine` was already in there)

Reply:
Merged that PR, saw all the new images published ([build link](https://github.com/rapidsai/ci-imgs/actions/runs/11804580918)), just pushed https://github.com/rapidsai/cudf/pull/17284/commits/4aa1d3c07e4492f02ff34b1365c7d2955e959df6 here removing this install.